### ISSUE TITLE
Handbook worked example uses Index.counts but the attribute is undocumented

### ISSUE THREAD

**Issue by jonkhler** [cusp-ai-oss/kUPS#25]
The handbook's worked \`md_lj\` example references \`.counts\` on an \`Index\`:

\`\`\`python
# docs/handbook.md:72-74
neighborlist_params = UniversalNeighborlistParameters.estimate(
    particles.data.system.counts, systems, lj_params.cutoff
)
\`\`\`

The Tables notebook is the canonical place where \`Index\` is introduced, and it documents \`.valid_mask\`, \`.indices\`, \`.keys\`, and \`_cls\`. It does not mention \`.counts\`. A reader hitting this in the handbook cannot tell what \`.counts\` is, what type it returns, or why it is the right first argument to \`UniversalNeighborlistParameters.estimate\`.

The Neighbor Lists notebook's own estimate example uses a different idiom for the same argument:

\`\`\`python
# docs/notebooks/neighborlist.md
n_per_sys = Table.arange(jnp.array([N]), label=SystemId)
params = UniversalNeighborlistParameters.estimate(n_per_sys, systems, cutoffs)
\`\`\`

So within the handbook two different expressions are used to construct the same input, and only one is explained.

### Suggested fix

Pick whichever of the two is authoritative and align the other:

- (a) If \`Index.counts\` is the intended public accessor, document it in \`docs/notebooks/tables.md\` alongside \`.valid_mask\`. A one-liner: \`\"Index.counts returns a Table[Key, Array] giving the number of times each key appears\"\` (or whatever the actual semantics are) plus a small example.
- (b) If the handbook's \`.counts\` form is a shorthand meant to read cleanly but not necessarily what users should type, rewrite the handbook snippet to match the Neighbor Lists notebook's \`Table.arange(...)\` idiom so there is exactly one documented way to build the argument.

### Scope

- \`docs/handbook.md\` and/or \`docs/notebooks/tables.md\` and/or \`docs/notebooks/neighborlist.md\`, depending on which resolution is chosen.

**Comment by n-gao** [comment]
The property is documented in the API docs.
